src/components/Coupon/index.jsx

```diff
@@ -56,7 +62,20 @@ const Coupon = ({ id, createdAt, amounts, compName, img, isTesting }) => {
       </Title>
       <div className="couponBody">
         <Card className="couponCard" hoverable style={{ width: 350 }}>
-          <img width={200} src={img} alt="logo" />
+          <a
```

Comment:
can we change this to a button ? not an a tag having an a tag means that it redirects it to another page, in case you don't want a button you can use span and add the property on click and edit the css so when someone hover it changes how mouse looks like